CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
-## [unreleased]
+## [v0.5.0] - 2025-01-08
 
 ### Added
```

Comment:
This section currently includes the bullet "RootBackend has new methods `get_order_statuses` and `set_order_status`" on main. It looks as though the latter was added for the `POST` endpoint that and has since been removed, so I think it should be updated to read "RootBackend has new method `get_order_statuses`".